collectors/meta_template/schema.json

```diff
@@ -0,0 +1,462 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Netdata collector single module meta",
+  "properties": {
+    "plugin_name": {
+      "type": "string",
+      "description": "Plugin name (e.g. apps.plugin, proc.plugin, go.d.plugin). It must match the name of the binary in the plugins.d directory."
```

Comment:
We do not have binaries neither  for `charts.d` nor for `python`:

```sh
bash-5.2$ file collectors/charts.d.plugin/charts.d.plugin
collectors/charts.d.plugin/charts.d.plugin: Bourne-Again shell script, ASCII text executable
bash-5.2$ file collectors/python.d.plugin/python.d.plugin
collectors/python.d.plugin/python.d.plugin: Bourne-Again shell script, ASCII text executable
```

Should we not use `executable` instead binaries?

Reply:
@Ancairon this is the unique disagreement  I have, except for this, everything looks good. :)